ocaml/xapi/xapi_host.ml

```diff
@@ -1679,8 +1679,12 @@ let sync_pif_currently_attached ~__context ~host ~bridges =
   let networks = Db.Network.get_all_records ~__context in
   let pifs = Db.PIF.get_records_where ~__context ~expr:(
       Eq (Field "host", Literal (Ref.string_of host))
-    ) in
-
+    ) |> List.filter (fun (pif, pif_rec) ->
```

Comment:
If you write `_pif` you'll avoid a warning for the unused `pif` variable

Reply:
I'd like to replace it with `_`